server/src/test-integration/java/com/thoughtworks/go/config/GoConfigMigrationIntegrationTest.java

```diff
@@ -1533,6 +1534,46 @@ public void shouldMigrateEverythingAsItIs_Migration125To126() throws Exception {
         XmlAssert.assertThat(migratedXml).and(expectedConfig).areIdentical();
     }
 
+    @Test
+    public void shouldRemoveSiteUrlsAsAnAttributesAndAddAsAChildElement_Migration126To127() throws Exception {
```

Comment:
can we also add another test, that the migration should do nothing when no siteUrl and secureSiteUrl is specified as server attribute?